cmd/operator-sdk/internal/genutil/k8s.go

```diff
@@ -34,6 +35,16 @@ import (
 func K8sCodegen() error {
 	projutil.MustInProjectRoot()
 
+	out, err := exec.Command("go", "env", "GOROOT").CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("failed to GOROOT from go env")
```

Comment:
```suggestion
		return fmt.Errorf("failed to get GOROOT from go env: %w", err)
```

Reply:
Done, sorry for the typo there! Good suggestion on including the `err`.